models/issue.go

```diff
@@ -504,57 +504,58 @@ func Issues(opts *IssuesOptions) ([]*Issue, error) {
 	}
 
 	sess := x.Limit(setting.IssuePagingNum, (opts.Page-1)*setting.IssuePagingNum)
+	sess.Alias("issue")
```

Comment:
Nevermind, the Alias function does not need to be called.  I will take that out.